### ISSUE TITLE
网易云客户端显示无法联网

### ISSUE THREAD

**Issue by willieee802** [UnblockNeteaseMusic/server#1124]
### Bug 描述

# 使用的docker版，命令如下
docker run -d -p 9090:8080 -p 9091:8081 -e JSON_LOG=true -e LOG_LEVEL=debug pan93412/unblock-netease-music-enhanced -e https://music.163.com -f 103.126.92.132
有试过去掉-f也不行

# nslookup music.163.com结果是：
nslookup music.163.com
Server:  dns.google
Address:  8.8.8.8

Non-authoritative answer:
Name:    overseasv4.music.ntes53.netease.com
Addresses:  103.126.92.132
          103.126.92.133
Aliases:  music.163.com
          music.ntes53.netease.com

# 代理设置
![image](https://github.com/UnblockNeteaseMusic/server/assets/113773438/9a9959e5-c7fe-4442-8f05-1867cabca790)

### 日志内容

2023-06-09 17:36:27 {"level":30,"time":1686296187909,"pid":1,"hostname":"235e1a818476","scope":"app","msg":"HTTP Server running @ http://0.0.0.0:8080"}
2023-06-09 17:36:27 {"level":30,"time":1686296187910,"pid":1,"hostname":"235e1a818476","scope":"app","msg":"HTTPS Server running @ http://0.0.0.0:8081"}
2023-06-09 17:36:30 {"level":20,"time":1686296190026,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:30 {"level":20,"time":1686296190233,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:35 {"level":20,"time":1686296195251,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197398,"pid":1,"hostname":"235e1a818476","scope":"server","url":"s6.music.126.net:443","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197849,"pid":1,"hostname":"235e1a818476","scope":"server","url":"ac.dun.163.com:443","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197851,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197961,"pid":1,"hostname":"235e1a818476","scope":"server","url":"st.music.163.com:443","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197968,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197995,"pid":1,"hostname":"235e1a818476","scope":"server","url":"ac.dun.163.com:443","msg":"TUNNEL"}
2023-06-09 17:36:37 {"level":20,"time":1686296197999,"pid":1,"hostname":"235e1a818476","scope":"server","url":"ac.dun.163.com:443","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199295,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199304,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199304,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199305,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199306,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199306,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199307,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199325,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199336,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199346,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199347,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199347,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199348,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199348,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199348,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199370,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199381,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199382,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199382,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199382,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199383,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199404,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199405,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199405,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199414,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199428,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199428,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199428,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199429,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199429,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199596,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199641,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199642,"pid":1,"hostname":"235e1a818476","scope":"server","url":"acstatic-dun.126.net:443","msg":"TUNNEL"}
2023-06-09 17:36:39 {"level":20,"time":1686296199648,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:40 {"level":20,"time":1686296200256,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:40 {"level":20,"time":1686296200615,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:40 {"level":20,"time":1686296200638,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:40 {"level":20,"time":1686296200889,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:41 {"level":20,"time":1686296201337,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:44 {"level":20,"time":1686296204045,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:45 {"level":20,"time":1686296205026,"pid":1,"hostname":"235e1a818476","scope":"server","url":"ac.dun.163.com:443","msg":"TUNNEL"}
2023-06-09 17:36:46 {"level":20,"time":1686296206028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:48 {"level":20,"time":1686296208178,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:50 {"level":20,"time":1686296210028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:56 {"level":20,"time":1686296216020,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:36:58 {"level":20,"time":1686296218184,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:02 {"level":20,"time":1686296222032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:08 {"level":20,"time":1686296228027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:08 {"level":20,"time":1686296228189,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:14 {"level":20,"time":1686296234024,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:18 {"level":20,"time":1686296238209,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:20 {"level":20,"time":1686296240021,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:26 {"level":20,"time":1686296246029,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:32 {"level":20,"time":1686296252030,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:38 {"level":20,"time":1686296258016,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:44 {"level":20,"time":1686296264033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:50 {"level":20,"time":1686296270021,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:37:56 {"level":20,"time":1686296276027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:02 {"level":20,"time":1686296282019,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:08 {"level":20,"time":1686296288028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:14 {"level":20,"time":1686296294019,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:18 {"level":20,"time":1686296298216,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:20 {"level":20,"time":1686296300034,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:26 {"level":20,"time":1686296306028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:32 {"level":20,"time":1686296312027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:38 {"level":20,"time":1686296318026,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:44 {"level":20,"time":1686296324032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:50 {"level":20,"time":1686296330023,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:38:56 {"level":20,"time":1686296336033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:02 {"level":20,"time":1686296342024,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:08 {"level":20,"time":1686296348025,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:14 {"level":20,"time":1686296354027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:18 {"level":20,"time":1686296358227,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:20 {"level":20,"time":1686296360034,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:26 {"level":20,"time":1686296366030,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:32 {"level":20,"time":1686296372026,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:38 {"level":20,"time":1686296378016,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:44 {"level":20,"time":1686296384027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:50 {"level":20,"time":1686296390028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:39:56 {"level":20,"time":1686296396023,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:02 {"level":20,"time":1686296402033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:08 {"level":20,"time":1686296408028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:14 {"level":20,"time":1686296414023,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:18 {"level":20,"time":1686296418236,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:20 {"level":20,"time":1686296420032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:26 {"level":20,"time":1686296426027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:32 {"level":20,"time":1686296432033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:38 {"level":20,"time":1686296438019,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:44 {"level":20,"time":1686296444023,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:50 {"level":20,"time":1686296450022,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:40:56 {"level":20,"time":1686296456033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:02 {"level":20,"time":1686296462029,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:08 {"level":20,"time":1686296468030,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:14 {"level":20,"time":1686296474028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:20 {"level":20,"time":1686296480027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:26 {"level":20,"time":1686296486022,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:32 {"level":20,"time":1686296492030,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:38 {"level":20,"time":1686296498015,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:44 {"level":20,"time":1686296504033,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:50 {"level":20,"time":1686296510028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:41:56 {"level":20,"time":1686296516032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:02 {"level":20,"time":1686296522027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:08 {"level":20,"time":1686296528035,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:14 {"level":20,"time":1686296534027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:20 {"level":20,"time":1686296540028,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:26 {"level":20,"time":1686296546022,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:32 {"level":20,"time":1686296552027,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:38 {"level":20,"time":1686296558029,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:44 {"level":20,"time":1686296564029,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:50 {"level":20,"time":1686296570032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:42:56 {"level":20,"time":1686296576022,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:02 {"level":20,"time":1686296582022,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:08 {"level":20,"time":1686296588032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:14 {"level":20,"time":1686296594031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:20 {"level":20,"time":1686296600031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:26 {"level":20,"time":1686296606032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:32 {"level":20,"time":1686296612025,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:38 {"level":20,"time":1686296618024,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:44 {"level":20,"time":1686296624032,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:50 {"level":20,"time":1686296630025,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:43:56 {"level":20,"time":1686296636023,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:02 {"level":20,"time":1686296642031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:08 {"level":20,"time":1686296648031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:14 {"level":20,"time":1686296654031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:20 {"level":20,"time":1686296660031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:26 {"level":20,"time":1686296666031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:32 {"level":20,"time":1686296672029,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}
2023-06-09 17:44:38 {"level":20,"time":1686296678031,"pid":1,"hostname":"235e1a818476","scope":"server","url":"localhost:8081","msg":"TUNNEL"}

### 网易云音乐版本号

2.10.10

### 操作系统

Windows 11

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by 1715173329** [comment]
duplicate of #1124

**Comment by willieee802** [comment]
> duplicate of #1124

这是这个帖子，quote错了？

**Comment by willieee802** [comment]
> duplicate of #1124

晕，装证书的时候，一个步骤选错了，已经弄好了，谢谢

**Comment by MarcuX009** [comment]
我也以为是挂了, 更新了最新版后, 再设置下代理就好. (感谢开发者)
![image](https://github.com/UnblockNeteaseMusic/server/assets/63788013/bc4f7d9b-40e1-463d-a5ef-70b38c245400)

**Comment by MangMax** [comment]
> -e [music.163.com](https://music.163.com) -f 103.126.92.132

请问你是怎么装的，我证书也安装了，但是日志也跟你一样，求教

**Comment by willieee802** [comment]
> > -e [music.163.com](https://music.163.com) -f 103.126.92.132
> 
> 请问你是怎么装的，我证书也安装了，但是日志也跟你一样，求教

这是开发者写的教程，确保每一步都一样
https://github.com/UnblockNeteaseMusic/server/discussions/426

我之前是选证书位置的时候，选成自动了
